### DISCUSSION TITLE
Simplified Coding API

### DISCUSSION THREAD

**Discussion by GalaxyGamingBoy** [simplified-coding/simplified-coding#128] [Ιδέες / Σχόλια]
The simplified coding API will be a quick solution to expose the lessons to a programmable interface for the public.

Prefix: `/api`
Suggested Routes:
- /api/courses
- /api/course/:lang
- /api/course/:lang/:lesson?content=bool

**Comment by GalaxyGamingBoy** [comment]
- /api/data/export
- /api/data/import
- /api/data/sync
- /api/data/remove

**Comment by odysseuskir** [comment]
Can you elaborate on this?
